tutorials/fiori-tools-mockserver-opa/fiori-tools-mockserver-opa.md

```diff
@@ -0,0 +1,222 @@
+
+    >It might help to use notepad to construct the following `url`.
+
+    Construct a new URL to your Agency value help metadata file by combining from your running application `https://xxxx.xxx` + Metadata Call
+
+    ![New Metadata File](NewMetadataFile.png)
+
+    You should see the new metadata document, save it to `webapp/localService/AgencyVHMetadata.xml`    
+
+3. Agency Value Help service URL (`urlBasePath`)        
+
+    `urlBasePath` - is the path you copied from your  *info server:ux-proxy* without the **$metadata**
+
+4. Putting it all together in a new `config.json` file.  
+
+    Right-click on `localService` folder and select `New File`. Create the `config.json` under the following path `webapp/localService/config.json`.
+
+
+5. In your `ui5-mock.yaml` file copy the configuration-service.
+
+    Depending on where you place your `config.json` adjust the file path for the metadata file in `metadataXmlPath` and the mock data in `mockdataRootPath`.
+
+    ![UI-Mock Copy](ConfigCopy.png)
+
+    ```json
+    [{
+      "urlBasePath": "/sap/opu/odata4/sap/zui_fe_travel_######_o4/srvd/sap/zui_fe_travel_######_o4/0001",
+      "name": "",
+      "metadataXmlPath": "./metadata.xml",
+      "mockdataRootPath": "./mockdata",
+      "generateMockData": true
+    }]
+    ```
+    Paste the code into your new `config.json` file.
+
+    >You will need to add parenthesis and double quotes since the format of the `.json` file is different then the `.yaml` file.
+
+
+    ![Config File](configfile.png)
+
+6. In your `ui5-mock.yaml` switch the configuration to use the `config.json` file that you created earlier.
+
+    >It is extremely important that the `yaml` file reflects hierarchy.
+
+    In order to do that you need to replace your `sap-fe-mockserver` code snippet with this:
+
+    ```yaml
+    name: sap-fe-mockserver
+    beforeMiddleware: fiori-tools-proxy
+    configuration:
+      mockFolder: webapp/localService
+    ```
+
+    !![UI5-Mock.yaml File](ui5mockyamlfile.png)
+
+7. Validate that your mock server still works by executing `start-mock` command under **Preview Application**   
+
+    >If you are running into issues, make sure your formatting of the `ui5-mock.yaml` is correct.
+
+    ![ui5-mock.yaml formatting](ui5mockyamlformatting.png)
+
+8. Adding the new configuration to your `config.json`
+    * `urlBasePath` - is the path you copied from your  *info server:ux-proxy* without the **$metadata**
+    * `metadataXmlPath` - reflects the path and the name of your new metadata file that was saved in the above step.
+
+    ```json
+    {
+        "urlBasePath": "/sap/opu/odata4/sap/zui_fe_travel_######_o4/srvd_f4/dmo/i_agency/0001;ps=%27srvd-zfe_travel_######-0001%27;va=%27com.sap.gateway.srvd.zfe_travel_######.v0001.et-zc_fe_travel_######.agencyid%27",
+        "name": "",
+        "metadataXmlPath": "./AgencyVHMetadata.xml",
+        "mockdataRootPath": "./mockdata",
+        "generateMockData": true
+    }]
```

Comment:
Is this intended?

Reply:
I think, the object shown in the coding fragment is the last one within an array. Since only a part of the array is shown here in the text, it looks as if it is wrong, but in the real config file it is correct.
Therefore I would tend to leave it here as it is.